e2e/workspace/BUILD.bazel

```diff
@@ -0,0 +1,3 @@
+load("@npm//:defs.bzl", "npm_link_all_packages")
+
+npm_link_all_packages(name = "node_modules")
```

Comment:
should also add this to `ci.yaml` so it doesn't bit rot & remote the `---SNIP---` from the old location

Reply:
will do in follow-up